### DISCUSSION TITLE
Updated Plex poster source and updating overlays

### DISCUSSION THREAD

**Discussion by pwsh** [meisnate12/Plex-Meta-Manager#1869] [Q&A]
For the announcement at [https://forums.plex.tv/t/movie-artwork-updates/867738](https://forums.plex.tv/t/movie-artwork-updates/867738) about updated and higher quality posters, is there anything that needs to be done to use the updated artwork?

Since PMM is manually changing the artwork, my assumption is the updated artwork will not be pulled down. To get the updated artwork, my assumption is we will need to change PMM to revert and remove the overlays. 

This process would be
1. Using pmm_overlay_reset.py from this process [https://metamanager.wiki/en/latest/pmm/scripts/overlay-reset/](https://metamanager.wiki/en/latest/pmm/scripts/overlay-reset/)
2. Force plex to update metadata [https://support.plex.tv/articles/200289306-scanning-vs-refreshing-a-library/](https://support.plex.tv/articles/200289306-scanning-vs-refreshing-a-library/)
3. Turn PMM back on and let it apply the overlays to the new posters (new items will automatically get new posters)

*edit to add the new posters are bad [https://www.reddit.com/r/PleX/comments/1agl92g/the_new_posters_are_almost_entirely_worse/](https://www.reddit.com/r/PleX/comments/1agl92g/the_new_posters_are_almost_entirely_worse/)

**Comment by chazlarson** [comment]
What announcement?  That's just the overlay-reset wiki page.

**Reply by pwsh** [reply]
apologies. Fixed the link. Copy and paste error.

**Reply by chazlarson** [reply]
All those links go to 404 pages.

**Reply by pwsh** [reply]
The link addresses were correct. Github just didn't modify the URL to reflect the link address so it was redirecting to this discussion. I corrected those.

**Reply by chazlarson** [reply]
Yers, I meant the links as clicked in this discussion item went to github 404 pages.

**Comment by chazlarson** [comment]
If you just want to remove overlays there should be no reason to use the reset script.

**Reply by pwsh** [reply]
I was trying to get to the least number of steps and the easiest option for a reset and update to use the new poster source as I am certain I will not be the only one with this question.

If I could provide quality steps that could easily be validated, that would simplify if others are looking to do the same.

It could be as simple as setting reapply overlays if internally Plex is still updating all their backend assets, but I haven't been able to confirm how they are changing their assets or determining what to update.
